docs/oras-auth-provider.md

```diff
@@ -0,0 +1,150 @@
+6. As the guide linked above shows, it's possible to use the AZ workload identity CLI or the regular az CLI to perform remaining setup. Following steps follow the AZ CLI.
+7. Create ACR AAD application: `az ad sp create-for-rbac --name "<APPLICATION_NAME>"`
+8. On Portal or AZ CLI, enable acrpull role to the AAD application for the ACR resource
+9. Use kubectl to add service account to cluster: 
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    azure.workload.identity/client-id: ${APPLICATION_CLIENT_ID}
+  labels:
+    azure.workload.identity/use: "true"
+  name: ${SERVICE_ACCOUNT_NAME}
+  namespace: ${SERVICE_ACCOUNT_NAMESPACE}
+EOF
+```
+10. From Azure Cloud Shell: 
+```
+export APPLICATION_OBJECT_ID="$(az ad app show --id ${APPLICATION_CLIENT_ID} --query objectId -otsv)"
+cat <<EOF > body.json
+{
+  "name": "kubernetes-federated-credential",
+  "issuer": "${SERVICE_ACCOUNT_ISSUER}",
+  "subject": "system:serviceaccount:${SERVICE_ACCOUNT_NAMESPACE}:${SERVICE_ACCOUNT_NAME}",
+  "description": "Kubernetes service account federated credential",
+  "audiences": [
+    "api://AzureADTokenExchange"
+  ]
+}
+EOF
+
+az rest --method POST --uri "https://graph.microsoft.com/beta/applications/${APPLICATION_OBJECT_ID}/federatedIdentityCredentials" --body @body.json
+```
+11. In the Pod spec, add the `serviceAccountName` property. Example Pod spec:
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: quick-start
+  namespace: <SERVICE_ACCOUNT_NAMESPACE>
+spec:
+  serviceAccountName: <SERVICE_ACCOUNT_NAME>
+  ...
+EOF
+```
+
+#### Ratify Auth Provider Configuration
+```
+"auth-provider": {
+    "name": "azure-wi"
+}
+```
+
+### 3. Kubernetes Secrets
+Ratify resolves registry credentials from [Docker Config Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets) in the cluster. Ratify considers kubernetes secrets in two ways. First, the configuration can specify a list of `secrets`. Each entry REQUIRES the `secretName` field. The `namespace` field MUST also be provided if the secret does not exist in the namespace Ratify is deployed in. The Ratify pod MUST be assigned the necessary read secret role in order to read the listed secrets from the namespace. Second, Ratify considers the `imagePullSecrets` specified in the service account associated with Ratify. The `serviceAccountName` field specifies the service account associated with Ratify. Ratify MUST be assigned a role to read the service account and secrets in the Ratify namespace.
+
+The Ratify helm chart contains a roles.yaml file with role assignments. If a namespace other than Ratify's namespace is provided, the user MUST add a new role binding to the cluster role so Ratify's service account can operate. 
```

Comment:
RBAC descriptions still seem a bit confusing to follow. I'm still struggling to find the best way to explain it. Any suggestions?

Reply:
I think we can break down into two section/paragraphs.  1. Kubernetes Secrets , 2. Image Pull secrets, then it will be clearer if some step is necessary for both methods, or only applicable to one.

Reply:
I separated it into 2 sections. I think it's a bit clearer now which permissions are need for each step.

Reply:
thanks it looks good! Teja mentioned earlier, lets add a hyper link to ratify roles.yaml .